pkgs/development/libraries/openssl/1.0.2.x.nix

```diff
@@ -18,7 +18,9 @@ stdenv.mkDerivation rec {
     sha1 = "d01d17b44663e8ffa6a33a5a30053779d9593c3d";
   };
 
-  patches = optional stdenv.isCygwin ./1.0.1-cygwin64.patch;
+  patches =
+    [ ./openssl-1.0.0d-windres.patch ]
```

Comment:
Would this be the right way to make the patch optional against specific cross compile targets?

```
optional (builtins.elem stdenv.cross.config ["i686-w64-mingw32" "x86_64-w64-mingw32"]) ./openssl-1.0.0d-windres.patch
```